src/mosqlimate/settings/base.py

```diff
@@ -129,6 +129,20 @@
 # 2 Factor Authentication (allauth)
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
 
+##
+STORAGE_DIR = Path(__file__).resolve().parent.parent.parent.parent
```

Comment:
Could we use BASE_DIR as the relative dir to the `STORAGE_DIR`? IMO it would be more intuitive as `BASE_DIR.parent`

Reply:
> Could we use BASE_DIR as the relative dir to the `STORAGE_DIR`? IMO it would be more intuitive as `BASE_DIR.parent`

Sure! We can use BASE_DIR as the relative directory for STORAGE_DIR. The initial idea of having STORAGE_DIR was to save space in the root directory ("/") by using the "/storage" partition. However, since the cache size is insignificant and stored within the repository, I'm already changing it to base_dir.
Thanks.